# 待機モードに移行(SCIBOT-Xのみ)
---

